src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/ciagmigration/CiagInductionMigrationService.kt

```diff
@@ -12,22 +12,34 @@ private val log = KotlinLogging.logger {}
  * API and database.
  */
 @Service
-class CiagInductionMigrationService(private val ciagWebClient: WebClient) {
+class CiagInductionMigrationService(
+  private val timelineLookupService: PrisonerInductionTimelineLookupService,
+  private val ciagWebClient: WebClient,
+) {
 
   fun migrateCiagInductions() {
     log.info { "Starting migration of CIAG Inductions from CIAG API to PLP API" }
 
     try {
-      val ciagInduction = ciagWebClient
-        .get()
-        .uri("/ciag/induction/A5077DY")
-        .retrieve()
-        .bodyToMono(CiagInductionResponse::class.java)
-        .block()
-
-      log.info { "Deserialised CIAG Induction: $ciagInduction" }
+      val prisonNumbers = timelineLookupService.getPrisonNumbersToMigrate()
+      log.info { "Identified ${prisonNumbers.size} Inductions to migrate" }
+
+      prisonNumbers.forEach {
```

Comment:
We're expecting to have a small number of Inductions to migrate, so this should be fine. Otherwise I'd probably have to call Sacintha's `/list` endpoint, but I've never tested it. Also, given it pulls back all the data for each Induction, I'm not convinced it's _that_ much more efficient anyway! 😄